PowerShell/JumpCloud Module/Tests/Public/Users/Get-JCUser.tests.ps1

```diff
@@ -395,12 +393,80 @@ Describe -Tag:('JCUser') "Get-JCUser 1.12" {
     It "Searches for a user by external_dn and external_source_type" {
 
         $Newuser = New-RandomUser -domain "deleteme" | New-JCUser
-        $Random1 = $(Get-Random)
-        $Random2 = $(Get-Random)
+        $Random1 = Get-Random
+        $Random2 = Get-Random
         $SetUser = Set-JCUser -Username $Newuser.username -external_source_type "$Random1" -external_dn "$Random2"
         $SearchUser = Get-JCUser -external_source_type "$Random1" -external_dn "$Random2"
         $RemoveUser = Remove-JCUser -UserID  $Newuser._id -force
         $SearchUser._id | Should -Be $Newuser._id
 
     }
+}
+
+Describe -Tag:('JCUser') "Case Insensitivity Tests" {
```

Comment:
Again, I feel like we should really break out the test for case insensitivity & special charcters